src/style.css

```diff
@@ -1 +1,83 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  font-family: "Roboto", sans-serif;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  max-width: 100%;
+  padding: 0 50px;
+  margin: 0 auto;
+}
+
+.logo {
+  padding-top: 5px;
+}
+
+.nav__list {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.nav {
+  display: flex;
+  align-items: stretch;
+}
+
+.nav__section {
+  list-style: none;
+  display: flex;
+  margin-left: 20px;
+}
+
+.nav__sectionApple--margin--0 {
+  list-style: none;
+  display: flex;
+  margin-left: 0;
+  margin-right: 0;
+}
+
+.link {
+  padding-bottom: 23px;
+  padding-top: 23px;
+  text-decoration: none;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
```

Comment:
Don't duplicate font-rules you can apply it to the html tag and all children will inherit it